.travis.yml

```diff
@@ -42,11 +42,11 @@ jobs:
         - pip install pyctdev && doit miniconda_install && pip uninstall -y doit pyctdev
         - export PATH="$HOME/miniconda/bin:$PATH" && hash -r
         - conda config --set always_yes True
-        - conda install -c pyviz "pyctdev>=0.5" pyctdev && doit ecosystem_setup
+        - conda install -c pyviz/label/dev "pyctdev>=0.5" pyctdev && doit ecosystem_setup
       before_script:
         - doit env_create --name=pyviz --python=$PYENV_VERSION
         - source activate pyviz
-        - travis_wait 20 doit develop_install $CHANS_DEV -o tests -o examples -o graphs -o geo -o indirect
+        - travis_wait 20 doit develop_install $CHANS_DEV -o recommended --all-extras
```

Comment:
I think you should remove `-o recommended` as it isn't doing anything. I think pyctdev should be giving an error for this, as there's no `recommended` set of extras defined in setup.cfg.

Reply:
I filed a bug report: https://github.com/pyviz/pyct/issues/73

Reply:
Thanks, so I should define no ``-o`` at all since ``--all-extras`` is already doing it?

Reply:
Getting an error with `develop_install -c pyviz/label/dev --all-extras`

Reply:
The main thing I'm confused about here is why ``-o examples`` doesn't work, in the tox.ini that defines all the correct dependencies. Is it getting confused about the ``examples`` entry in `extras_require`?

Reply:
> Thanks, so I should define no -o at all since --all-extras is already doing it?

Right

> Getting an error with doit develop_install $CHANS_DEV --all-extras

The error being the following? (I can't read as fast as you can code 😂) 

```
PackagesNotFoundError: The following packages are not available from current channels:
  - fastparquet[version='>=0.2.1']
Current channels:
  - https://conda.anaconda.org/pyviz/label/dev/linux-64
  - https://conda.anaconda.org/pyviz/label/dev/noarch
  - https://repo.anaconda.com/pkgs/main/linux-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/free/linux-64
  - https://repo.anaconda.com/pkgs/free/noarch
  - https://repo.anaconda.com/pkgs/r/linux-64
  - https://repo.anaconda.com/pkgs/r/noarch
```

Or something else?

Reply:
Haha, yes, I just didn't look closely enough thanks, that makes sense now. If you can explain to me why ``-o examples`` didn't work (see https://travis-ci.org/pyviz/pyviz/builds/486033035), i.e. didn't install the indirect dependencies, I think I'll finally understand how some of this stuff works.